emacs/cquery.el

```diff
@@ -65,6 +65,13 @@
   :type 'directory
   :group 'cquery)
 
+(defcustom cquery-indexer-threads
+  0
+  "Number of threads cquery will use to index each project. When 0, use
+one thread for each available core."
```

Comment:
This is not quite 1-1, it is something like 70% or so

Reply:
Er, yeah, just read the package.json documentation. `-threads` looks like a bit of a misnomer, too… I'll bring the docstring in line with the vscode extension.